code/modules/mob/living/carbon/alien/humanoid/humanoid_defense.dm

```diff
@@ -39,7 +39,7 @@
 					to_chat(M, "<span class='warning'>Your punch misses [src]!</span>")
 
 			if ("disarm")
-				if (!(mobility_flags & MOBILITY_STAND))
+				if (body_position == STANDING_UP)
```

Comment:
Should be laying

Reply:
`[M] pushes you down!` and `You hear aggressive shuffling followed by a loud thud!` seem to indicate the target needs to be standing up.